libglusterfs/src/common-utils.c

```diff
@@ -642,6 +642,123 @@ gf_dnscache_entry_deinit(struct dnscache_entry *entry)
     GF_FREE(entry);
 }
 
+/**
+ * gf_dns_addr_cache_deinit -- cleanup resources used by struct dnscache
+ */
+void
+gf_dns_addr_cache_deinit(struct dnscache *cache)
+{
```

Comment:
This is using internal dict_t structure. This should never be accessed from outside `libglusterfs/src/dict.c`.

Reply:
done